src/MauiScreenTime/Services/IUsageStatsService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MauiScreenTime.Data;
+
+namespace MauiScreenTime.Services
+{
+    public interface IUsageStatsService
+    {
```

Comment:
Solid, if a component will need to depend on this service it'll be unit testable by mocking this